pkgs/development/libraries/qtspell/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  doxygen,
+  enchant,
+  glib,
+  kdePackages,
```

```suggestion
  qtbase,
  qttool,
```
Packages defined in qt6-packages.nix should not use kdePackages/qt6Packages, just use qttools which is the qt 6 version